HW1/P5/P5.txt

```diff
@@ -1 +1,160 @@
-# Your discussion here
+Note that helper functions are stored in the "network_commands" python module in the directory above this one.
+I wanted to place all of my network code in the same place.
+
+###### Shortest path #######
+
+The shortest path between 'Harvard_University' and 'Kevin_Bacon' is of length 3. One example is
```

Comment:
I believe the question asks for the shortest path from Kevin Bacon to Harvard University and the shortest path from Harvard University to Kevin Bacon.  Since this is a directed graph, those paths are not necessarily the same length.  Here you have found a shortest path from Harvard University to Kevin Bacon (which has length 3).  But you haven't stated the length of the shortest path from Kevin Bacon to Harvard University (which has length 2)